website/docs/components/form/super-select/partials/code/component-api.md

```diff
@@ -0,0 +1,280 @@
+## Component API
+
+Super Select consists of two main components, `SuperSelect::Single` and `SuperSelect::Multiple`.
+
+Each of these main components have two different variants with their own APIs:
+
+- `Form::SuperSelect::Single::Base` or `Form::SuperSelect::Multiple::Base`—the base component with just the Super Select control.
+- `Form::SuperSelect::Single::Field` or `Form::SuperSelect::Multiple::Field`—the field parent component with the Super Select control, label, helper text, and error messaging (in a wrapping container).
+
+The Single and Multiple base components have nearly identical APIs to the 8.1.0 version of the [ember-power-select](https://ember-power-select.com/) addon.
+
+While the [ember-power-select documentation](https://ember-power-select.com/docs) has the full list of supported options for the base components, we’ve included the most common ones here for your convenience as well as the options for which default values have been overridden.
```

Comment:
I think Kristin already mentioned this on Slack – the `ember-power-select` documentation doesn't support direct links, so this would result in a 404. We'd probably have to use the root URL when referring to the docs (https://ember-power-select.com) and hope the consumers will find the relevant docs.

Reply:
Updated these